app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.length)
```

Comment:
- [ ] Nice job on implementing the nesseccary codes, however,  the update method updates the posts or comments, and like each time you push something new to your database and this is good but you can use increments instead to push new posts or comments or like with ease.

Reply:
Thank you for your valuable feedback.